source/grid/tria.cc

```diff
@@ -2054,8 +2054,8 @@ namespace internal
           for (i = needed_lines.begin(); line != triangulation.end_line();
                ++line, ++i)
             {
-              line->set(internal::TriangulationImplementation::TriaObject<1>(
-                i->first.first, i->first.second));
+              line->set(internal::TriangulationImplementation::TriaObject(
+                {i->first.first, i->first.second}));
```

Comment:
If you get rid of `TriaObject`, you could let the `TriaAccessor::set()` function just take the `std::initializer_list` directly.